pkgs/applications/office/jabref/default.nix

```diff
@@ -2,11 +2,13 @@
 , zlib , libX11, libXext, libXi, libXrender, libXtst, libGL, alsa-lib, cairo, freetype, pango, gdk-pixbuf, glib }:
 
 stdenv.mkDerivation rec {
-  version = "5.1";
+  version = "5.3";
   pname = "jabref";
 
-  src = fetchurl {
-    url = "https://github.com/JabRef/jabref/releases/download/v${version}/JabRef-${version}-portable_linux.tar.gz";
+  src = fetchFromGiHub {
```

Comment:
You're changing the `src` from a builded version (`...portable_linux.tar.gz)` to source code and not adjusting the rest of the expression.

Either adapt the expression to build jabref from source or follow the existing pattern and just update `version` and `sha256`.